test/integration/features/addon-xpaas.feature

```diff
@@ -19,18 +19,18 @@ which are then available in OpenShift to the user.
 
   Scenario Outline: User deploys, checks out and deletes several templates from XpaaS imagestream
    Given Minishift has state "Running"
-    When executing "oc new-app <template-name>" succeeds
-     And executing "oc set probe dc/<service-name> --readiness --get-url=http://:8080<http-endpoint>" succeeds
+    When executing "oc new-project <project-name>" succeeds
+    Then executing "oc new-app <template-name>" succeeds
```

Comment:
-1, we should not remove the `oc set` command as it creates the  probe

Reply:
Done, added back